### PR TITLE
Suggest UAs disconnect when a BluetoothDevice is gc'ed.

### PR DESCRIPTION
**Pull request by jyasskin**
Preview at https://rawgit.com/jyasskin/web-bluetooth-1/disconnect-on-gc/index.html.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2129,6 +2133,20 @@ spec: webidl
...
     <p>
+      When no ECMAScript code can
+      observe an instance of {{BluetoothRemoteGATTServer}} <var>server</var> anymore,
+      the UA SHOULD run <code><var>server</var>.{{BluetoothRemoteGATTServer/disconnect()}}</code>.
+      <span class="note">
+        Because {{BluetoothDevice}} instances are stored in
+        <code>navigator.bluetooth.{{Bluetooth/[[deviceInstanceMap]]}}</code>,
+        this can't happen at least until navigation releases the global object or
+        closing the tab or window destroys the <a>browsing context</a>.
+        Disconnecting on garbage collection ensures that
```
Comment (g-ortuno) [inline index.bs]: These two sentences seem orthogonal. I think they should be two separate notes.

Comment (jyasskin) [inline index.bs]: Sure. All in 1 paragraph or in 3 different paragraphs?

Comment (g-ortuno) [inline index.bs]: One works for me. I just saw both sentences be in the same note and kept trying to find the relation between them.

**Comment by g-ortuno** [comment]
lgtm

File: index.bs (hunk 1 above)
Comment (jyasskin) [inline index.bs]: Done.
